lib/web/mage/tabs.js

```diff
@@ -102,28 +102,49 @@ define([
          * @private
          */
         _processPanels: function () {
-            this.contents = this.element.find(this.options.content);
+            var isNotNested = this._isNotNested.bind(this);
 
-            this.collapsibles = this.element.find(this.options.collapsibleElement);
+            this.contents = this.element
+                .find(this.options.content)
+                .filter(isNotNested);
+
+            this.collapsibles =  this.element
+                .find(this.options.collapsibleElement)
+                .filter(isNotNested);
 
             this.collapsibles
                 .attr('role', 'presentation')
                 .parent()
                 .attr('role', 'tablist');
 
-            this.headers = this.element.find(this.options.header);
+            this.headers = this.element
+                .find(this.options.header)
+                .filter(isNotNested);
 
             if (this.headers.length === 0) {
                 this.headers = this.collapsibles;
             }
-            this.triggers = this.element.find(this.options.trigger);
+            this.triggers = this.element
+                .find(this.options.trigger)
+                .filter(isNotNested);
 
             if (this.triggers.length === 0) {
                 this.triggers = this.headers;
             }
             this._callCollapsible();
         },
 
+        /**
+         * @param {HTMLElement} elem
```

Comment:
Please adjust the JSDoc as the `index` is missing.

Reply:
Can we also get some description what this method does?

Reply:
Thanks @krzksz! Index param added, description added.

Reply:
There's still `elem` instead of `element`.

Reply:
Fixed.